jcl/source/common/JclSysUtils.pas

```diff
@@ -4528,7 +4528,7 @@ finalization
   {$ENDIF UNITVERSIONING}
   {$IFDEF MSWINDOWS}
   {$IFDEF THREADSAFE}
-  // The user must release shared memory blocks himself. We don't clean up his
+  // The user must release shared memory blocks himself. We don't clean up their
```

Comment:
I'm not a native speaker but it feels weird having "himself" followed by "their".